Tests/Tests/STPEphemeralKeyManagerTest.m

```diff
@@ -32,12 +31,16 @@ - (void)setUp {
     self.apiVersion = @"2015-03-03";
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
+
 - (id)mockKeyProviderWithKeyResponse:(NSDictionary *)keyResponse {
     XCTestExpectation *exp = [self expectationWithDescription:@"createCustomerKey"];
     id mockKeyProvider = OCMProtocolMock(@protocol(STPEphemeralKeyProvider));
     OCMStub([mockKeyProvider createCustomerKeyWithAPIVersion:[OCMArg isEqual:self.apiVersion]
                                                   completion:[OCMArg any]])
     .andDo(^(NSInvocation *invocation) {
+        [invocation retainArguments];
```

Comment:
why is this needed?

Reply:
There is a bug using `OCMock` that this avoids: https://github.com/erikdoe/ocmock/issues/147

Reply:
👍 a comment referencing this would be helpfule

Reply:
Let me add this